projects/packages/masterbar/src/nudges/bootstrap.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Bootstrap file for the nudges.
+ *
+ * @package jetpack-masterbar
+ */
+
+namespace Automattic\Jetpack\Masterbar;
+
+use Automattic\Jetpack\Status;
+use Automattic\Jetpack\Status\Host;
+
+/**
+ * The WP_Customize_Control core class is loaded only on customize_register.
+ *
+ * @param \WP_Customize_Manager $customize_manager Core customize manager.
+ */
+function register_css_nudge_control( \WP_Customize_Manager $customize_manager ) {
+	$domain = ( new Status() )->get_site_suffix();
```

Comment:
Are we missing the `require` like the ones we have in the [module file](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/modules/masterbar/nudges/bootstrap.php#L19-L28) or are we autoloading them somewhere else?

Reply:
Yes, composer takes care of this for us with:
```
"autoload": {
		"classmap": [
			"src/"
		]
	},
```

You can check the `autoload_classmap.php` Composer generates under 'masterbar/vendor/composer'